pdns/ixfrutils.cc

```diff
@@ -110,8 +110,9 @@ void writeZoneToDisk(const records_t& records, const DNSName& zone, const std::s
   fprintf(fp, "$ORIGIN %s\n", zone.toString().c_str());
   for(const auto& outer : {soarecord, records, soarecord} ) {
     for(const auto& r: outer) {
-      fprintf(fp, "%s\tIN\t%s\t%s\n",
+      fprintf(fp, "%s\t%i\tIN\t%s\t%s\n",
```

Comment:
`"%i"` is for an `int`, `d_ttl` is a `uint32_t` so we should use `"%"PRIu32`, which requires:
```
#include <cinttypes>
```